packages/e2e-tests/tests/user-invitation/invitation.spec.js

```diff
@@ -0,0 +1,64 @@
+const { AdminCreateUserPage } = require('../../fixtures/admin/create-user-page');
+const { publicTest, expect } = require('../../fixtures/index');
+const { subDays } = require('date-fns');
+const { client } = require('../../fixtures/postgres-client-config');
+const { v4: uuidv4 } = require('uuid');
+
+publicTest.describe('Accept invitation page', () => {
+  publicTest.beforeAll(async () => {
+    await client.connect();
+  });
+
+  publicTest.afterAll(async () => {
+    await client.end();
+  });
+
+  publicTest('should not be able to set the password if token is empty', async ({ acceptInvitationPage }) => {
+    await acceptInvitationPage.open('');
+    await acceptInvitationPage.excpectSubmitButtonToBeDisabled();
+    await acceptInvitationPage.fillPasswordField('something');
+    await acceptInvitationPage.excpectSubmitButtonToBeDisabled();
+  });
+
+  publicTest('should not be able to set the password if token is expired', async ({ acceptInvitationPage, page }) => {
+    const dateToday = new Date().toISOString().replace('T', ' ');
+    const expiredTokenDate = subDays(dateToday, 3).toISOString().replace('T', ' ');
+    const expiredToken = uuidv4();
+
+    const adminCreateUserPage = new AdminCreateUserPage(page);
+    adminCreateUserPage.seed(Math.ceil(Math.random() * Number.MAX_SAFE_INTEGER));
+    const user = adminCreateUserPage.generateUser();
+
+    const queryRole = {
+      text: 'SELECT * FROM roles WHERE name = $1',
+      values: ['Admin']
+    };
+
+    try {
+      const queryRoleIdResult = await client.query(queryRole);
+      expect(queryRoleIdResult.rowCount).toEqual(1);
+
+      const insertUser = {
+        text: 'INSERT INTO users (email, created_at, updated_at, full_name, role_id, status, invitation_token, invitation_token_sent_at) VALUES ($1, $2, $3, $4, $5, $6, $7, $8)',
```

Comment:
We can reduce the columns to `(email, full_name, role_id, status, invitation_token, invitation_token_sent_at)` as the others are either auto-filled or not needed.